evergreen.yml

```diff
@@ -182,7 +183,8 @@ tasks:
 ##                ⚡️ Task Groups ⚡️
 
 task_groups:
-- name: t_test
+- &test_group
```

Comment:
Calling this `test_group` is probably going to confuse someone. How about `compile_and_run_tests`?

Reply:
Yeah, that's a good idea. Thanks!